core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -191,7 +191,7 @@ query.timeout.sec=60
 # number of attempts to get TableReader
 #cairo.reader.pool.max.segments=5
 
-# timeout in microseconds when attempting to get BitmapIndexReaders
+# timeout in millis when attempting to get BitmapIndexReaders
 #cairo.spin.lock.timeout=1000000
```

Comment:
The default should be `1000` - this file is not impacting the operation.

Reply:
should be `1000`, good spot 👍